ehitchhiking-ui/src/app/shared/components/rating/rating.component.sass

```diff
@@ -11,12 +10,12 @@
 .rating:not(:checked) > label
   float: right
   width: 1rem
-  padding: 0.1rem
+  padding: $default-offset * 0.1
   overflow: hidden
   white-space: nowrap
   cursor pointer
   font-size: $button
-  line-height: 1rem
+  line-height: $default-offset
```

Comment:
you should not compute line-height using $default-offset. $default-offset should be used for paddings, margings